### ISSUE TITLE
loss问题

### ISSUE THREAD

**Issue by zhuchangjiang** [yusanshi/NewsRecommendation#15]
your code:
y = torch.zeros(len(y_pred)).long().to(device)
loss = criterion(y_pred, y)

这个y=[0,0,0,0,0,0,0,0] 
这个y是不是错了

**Comment by zhuchangjiang** [comment]
y不是 应该这样么[1,0,0,0,0,0,0,0]

**Comment by zhuchangjiang** [comment]
懂了。

**Comment by WeSeewy** [comment]
> 

我也有这个疑问，可以分享一下你的理解吗？感谢！
